src/planning/strategies/tests/coalesce-recipes-test.ts

```diff
@@ -200,32 +200,46 @@ describe('CoalesceRecipes', () => {
       `;
     };
 
-    await doCoalesceRecipes(await parseManifest({fateA: 'map'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'map', fateB: 'map'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'map', fateB: 'use'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'map', fateB: '?'}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'map', fateB: 'use', outThingB: true}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'map', fateB: 'copy'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'copy'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'copy', fateB: 'map'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'copy', fateB: 'copy'}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'copy', fateB: 'create'}));
-
-    await doCoalesceRecipes(await parseManifest({fateA: 'use', fateB: 'use'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'use', fateB: 'use', outThingB: true}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'use', fateB: '?'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'use', fateB: '?', outThingB: true}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'use', fateB: 'create'}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'use', fateB: 'map'}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'use', fateB: 'copy'}));
-
-    await doCoalesceRecipes(await parseManifest({fateA: 'create'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'create', fateB: '?'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'create', fateB: 'use'}));
-    await doCoalesceRecipes(await parseManifest({fateA: 'create', fateB: 'use', outThingB: true}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'create', fateB: 'create'}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'create', fateB: 'map'}));
-    await doNotCoalesceRecipes(await parseManifest({fateA: 'create', fateB: 'copy'}));
+    const expectSuccessData = [
```

Comment:
This change can be broken out from the rest, but it made it much easier to see which group of tests where supposed to succeed.